resources/prometheus/unit_tests/RHACSCentralSLISLO.yaml

```diff
@@ -0,0 +1,143 @@
+rule_files:
+  - /tmp/prometheus-rules-test.yaml
+
+evaluation_interval: 1m
+
+group_eval_order:
+  - rhacs-central.sli
+  - rhacs-central.slo
+  - rhacs-central.alerts
+
+tests:
+  # Central availability error budget exhaustion - 90%
+  - interval: 5m
+    input_series:
+      # 15m downtime due to pod not ready.
+      - series: kube_pod_container_status_ready{container="central", pod="central-test1", namespace="rhacs-test1"}
+        values: "1+0x300 0 0 0 1+0x100"
+      - series: grpc_server_handled_total{job="central", grpc_type="unary", grpc_code="OK", namespace="rhacs-test1"}
+        values: "1+1x400"
+      - series: grpc_server_started_total{job="central", grpc_type="unary", namespace="rhacs-test1"}
+        values: "1+1x397 399+2x1"
+      - series: http_incoming_requests_total{job="central", code="200", namespace="rhacs-test1"}
+        values: "4+4x400"
+      # 5m downtime due NOK or 5xx responses.
+      - series: grpc_server_handled_total{job="central", grpc_type="unary", grpc_code="NOK", namespace="rhacs-test1"}
+        values: "0+0x397 0+1x1"
+      - series: http_incoming_requests_total{job="central", code="500", namespace="rhacs-test1"}
+        values: "0+0x397 0+4x1"
+    alert_rule_test:
+      - eval_time: 100m
+        alertname: Central availability error budget exhaustion - 90%
+        exp_alerts: []
+      - eval_time: 2000m
+        alertname: Central availability error budget exhaustion - 90%
+        exp_alerts:
+          - exp_labels:
+              alertname: Central availability error budget exhaustion - 90%
+              service: central
+              severity: critical
+              team: appsre
+              namespace: rhacs-test1
+            exp_annotations:
+              message: "High availability error budget exhaustion for central. Current exhaustion: 100.2%."
+              dashboard: "https://grafana.app-sre.devshift.net/d/T2kek3H9a/central-slos?orgId=1&from=now-28d&to=now"
```

Comment:
These are the dashboard/runbooks for the acs-fleet-manager. I'd suggest removing the dashboard link, and creating an entry in https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks for the alert runbook. Also rename `runbook` to `sop_url` for consistency with the other data plane alerts.